tests/applications/test_junebug_message_api.py

```diff
@@ -0,0 +1,667 @@
+
+#         worker = yield self.get_worker({
+#             'message_rate_bucket': 1.0,
+#         })
+
+#         event = TransportEvent(
+#             event_type='nack',
+#             nack_reason='bad message',
+#             user_message_id='msg-21',
+#             timestamp='2015-09-22 15:39:44.827794')
+#         yield worker.consume_nack(event)
+
+#         clock.advance(1)
+
+#         self.assertEqual((yield worker.message_rate.get_messages_per_second(
+#             'testtransport', 'rejected', 1.0)), 1.0)
+
+#     @inlineCallbacks
+#     def test_delivery_succeeded_event_rates(self):
+#         '''Delivered delivery reports should increase the delivery_succeeded
+#         event rates.'''
+#         clock = self.patch_message_rate_clock()
+
+#         worker = yield self.get_worker({
+#             'message_rate_bucket': 1.0,
+#         })
+
+#         event = TransportEvent(
+#             event_type='delivery_report',
+#             user_message_id='msg-21',
+#             delivery_status='delivered',
+#             timestamp='2015-09-22 15:39:44.827794')
+#         yield worker.consume_delivery_report(event)
+
+#         clock.advance(1)
+
+#         self.assertEqual((yield worker.message_rate.get_messages_per_second(
+#             'testtransport', 'delivery_succeeded', 1.0)), 1.0)
+
+#     @inlineCallbacks
+#     def test_delivery_failed_event_rates(self):
+#         '''Failed delivery reports should increase the delivery_failed
+#         event rates.'''
+#         clock = self.patch_message_rate_clock()
+
+#         worker = yield self.get_worker({
+#             'message_rate_bucket': 1.0,
+#         })
+
+#         event = TransportEvent(
+#             event_type='delivery_report',
+#             user_message_id='msg-21',
+#             delivery_status='failed',
+#             timestamp='2015-09-22 15:39:44.827794')
+#         yield worker.consume_delivery_report(event)
+
+#         clock.advance(1)
+
+#         self.assertEqual((yield worker.message_rate.get_messages_per_second(
+#             'testtransport', 'delivery_failed', 1.0)), 1.0)
+
+#     @inlineCallbacks
+#     def test_delivery_pending_event_rates(self):
+#         '''Pending delivery reports should increase the delivery_pending
+#         event rates.'''
+#         clock = self.patch_message_rate_clock()
+
+#         worker = yield self.get_worker({
+#             'message_rate_bucket': 1.0,
+#         })
+
+#         event = TransportEvent(
+#             event_type='delivery_report',
+#             user_message_id='msg-21',
+#             delivery_status='pending',
+#             timestamp='2015-09-22 15:39:44.827794')
+#         yield worker.consume_delivery_report(event)
+
+#         clock.advance(1)
+
+#         self.assertEqual((yield worker.message_rate.get_messages_per_second(
+#             'testtransport', 'delivery_pending', 1.0)), 1.0)
+
+## Message/event forwarding over AMQP. Do we actually use this anywhere?
```

Comment:
This used to be used when we used Junebug to configure our USSD and SMS. USSD application was implemented using jssandbox, which got it's messages from AMQP. This is no longer used, as we now prefer to use k8s config instead of the Junebug API to configure the USSD channels.